src/plugins/intel_cpu/src/nodes/scaled_attn.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include <ie_common.h>
+#include <node.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace ov {
+namespace intel_cpu {
+namespace node {
+
+enum KernelTypes { KT_REF, KT_ONEDNN, KT_MLAS};
```

Comment:
This enum should be part of the ScaledDotProductAttention class.
Otherwise you introduce very generic enum inside intel_cpu::node namespace which can easily conflict with other nodes.

Reply:
Moved into the class but failed to make the enum as private field. It seems that the type of a template parameter should not be inside a class private area.